### PR TITLE
Roll fattest.databases into fattest.simplicity

### PR DESCRIPTION
**Pull request by jhanders34**
- Comment in fattest.databases bnd.bnd file indicates that the project can be rolled into fattest.simplicity when Java 8 is the minimum Java level.  That has been the now for many months.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaeeClient-9.0/io.openliberty.jakartaeeClient-9.0.feature
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaeeClient-9.0/io.openliberty.jakartaeeClient-9.0.feature:24]: Is this change related or included by accident?

Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jakartaeeClient-9.0/io.openliberty.jakartaeeClient-9.0.feature:24]: Yes

**Comment by jhanders34** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Review by KyleAure — CHANGES_REQUESTED** [review]
Wait until #15381 and #15387 are merged. Otherwise, we'll have conflicting test container versions.  
Otherwise, changes look great.  Thanks for the help Jared #:+1:

**Review by KyleAure — CHANGES_REQUESTED** [review]
Just some minor change requests

File: dev/fattest.simplicity/bnd.bnd
Comment (KyleAure) [inline dev/fattest.simplicity/bnd.bnd:102]: Update test containers to 1.15.1 and ducktape to 1.0.8

File: dev/fattest.simplicity/bnd.bnd
Comment (KyleAure) [inline dev/fattest.simplicity/bnd.bnd:42]: Can this be put under `componenttest.containers.database` instead?  that way it shows up under the other container focused package.

**Review by KyleAure — CHANGES_REQUESTED** [review]

File: dev/fattest.simplicity/bnd.bnd
Comment (jhanders34) [reply dev/fattest.simplicity/bnd.bnd]: I updated them.  I had to merge everything so it was necessary.  I also updated the other places that still had 1.15.0 and 1.0.7

File: dev/fattest.simplicity/bnd.bnd
Comment (jhanders34) [reply dev/fattest.simplicity/bnd.bnd:42]: I just did alphabetical order to match the rest of the exports as to order.

**Review by KyleAure — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Pull request merged**
